### DISCUSSION TITLE
Repeated wake word activation.

### DISCUSSION THREAD

**Discussion by mushie4282** [OpenVoiceOS/ovos-buildroot#209] [Q&A]
I disabled "hey mycroft" hotwords and implemented my own. After rebooting, OVOS continuously activates and transcribed random words. Does anyone know the diagnosis for this issue? 

Here is my GitHub that links my trained wake word models and user config files used.
- https://github.com/SeniorDesign-RC-LB/wakeWordModels/tree/main?tab=readme-ov-file

I am running Raspberry Pi 4B 2GB with RaspOVOS img (headless).

**Comment by builderjer** [comment]
Are you using openwakeword?

There is a `threshold` value you can play with.

Check the [readme](https://github.com/OpenVoiceOS/ovos-ww-plugin-openWakeWord/blob/dev/readme.md) for more details

**Reply by mushie4282** [reply]
Yes, I'm using openWakeWord. After changing the threshold, I'm still coming across the same problem. I've attached my issue from viewing the voice log.

![Screenshot 2024-01-20 183741](https://github.com/OpenVoiceOS/ovos-buildroot/assets/106356342/5b0afafb-489b-49e0-bdd3-c3befb550bf3)

**Reply by JarbasAl** [reply]
it seems like the trained models just are not very good

**Comment by mushie4282** [comment]
@JarbasAl, probably. 
I wanted to post this discussion to see if anyone else is running into the same issue.  Does anyone have tips or advice on training wake word models on Window machines?

**Comment by mushie4282** [comment]
**Update**
I swapped my model to a pretrained model (android.tflite) to use with `ovos-ww-plugin-precise-lite`. https://github.com/OpenVoiceOS/precise-lite-models/tree/master/wakewords/en
I'm still having the same issue, where continuous wake word activation occurs.
![image](https://github.com/OpenVoiceOS/ovos-buildroot/assets/106356342/e9b98d04-9a8c-4376-8162-2ff7a496d0ee)

**Reply by builderjer** [reply]
It sounds like the model was trained with synthetic voices. You might try changing some settings
